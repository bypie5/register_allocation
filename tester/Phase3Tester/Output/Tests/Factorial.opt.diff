--- SelfTestCases/Factorial.opt.out	2011-04-28 14:55:20.000000000 -0700
+++ ./Output/Tests/Factorial.opt.out	2020-02-19 16:01:53.000000000 -0800
@@ -1 +1 @@
-3628800
+Program has no "Main" function.
