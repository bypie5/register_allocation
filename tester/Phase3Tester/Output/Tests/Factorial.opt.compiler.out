func Main [in 0, out 0, local 0]
    $s0 = 
    PrintIntS($s0)
    ret
func Fac.ComputeFac [in 0, out 0, local 0]
    $s2 = LtS($s0 1)
    $s0 = 1
    $s0 = Sub($s1 1)
    $s3 = 
    $s0 = MulS($s1 $s3)
    ret

